stylesheet.css

```diff
@@ -0,0 +1,275 @@
+* {
+    box-sizing: border-box;
+}
+body {
+    margin: 0;
+    font-family: 'Poppins', sans-serif;
+    line-height: 1.6;
+}
+html{
+    scroll-behavior: smooth;
+}
+/* Nav bar styling */
+
+nav{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    padding: 20px;
+    background-color: #d1e0ef;
+    text-transform: uppercase;
+}
+.topnav{
+    padding: 12px;
+}
+nav a{
+    margin: 10px;
+}
+nav a:hover{
+    background-color:#fae3e5;
+    border-radius: 25px;
+    padding: 12px;
+}
+a:hover{
+    color: #e2431b;
+}
+/* Header styling  */
+header{
+    background-image: url("Images/Background.jpg");
+    height: 750px;
+    text-align: center;
+}
+.intro{
+    display: grid;
+    grid-template-columns: 60% 40%;
+    padding-top: 100px;    
+}
+.intro_item{
+    margin: 30px;
+    align-self: center;
+}
+.intro_item.intro_text{
+    background-color: #FFFFFF ;
+    padding: 100px 40px;
+    margin-left: 150px;
+    text-align: left;
+}
+.profile{
+    object-fit: cover;
+    border-radius: 50%;
+    height: 300px;
+    width: 300px;
+    display: block;
+    margin: auto;
+}
+p{
+    padding-bottom:10px;
+}
+/* header button styling */
+.header_button{
+    display: inline-block;
+    font-size: 20px;
+    border-radius: 45px;
+    border-style: none;
+    background-color: #FFFFFF;
+    padding: 30px;
+}
+.jump_to_contact{
+    text-decoration: none;
+}
+/* My work section styling */
+.my_work{
+    display: grid;
+    grid-template-columns: 40% 60%;
+    padding-top: 100px;
+    background-color: #d1e0ef; 
+}
+.project_text{
+    padding: 40px;
+    margin-left: 50px;
+}
+.project_image img{
+   height: 400px;
+   padding: 30px;
+   display: block;
+   margin-left: auto;
+   margin-right: auto;
+}
+.my_work_two{
+    display: grid;
+    grid-template-columns: 60% 40%;
+    padding-top: 100px;
+    background-color: #d1e0ef;
+}
```

Comment:
You could make this a bit DRY-er.

If you had a base class of `.my_work` that gets assigned to both of your work sections, and then a second class of `.my_work_reverse` that does nothing apart from override the columns to 60/40 when it has both `.my_work` and `.my_work_reverse`.

That'll eliminate the need to repeat all these rules

Reply:
nice!